app/views/admin/api/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="admin">
+  <h2 class="heading"><%= ts('API Tokens') %></h2>
+
+  <ul class="navigation actions" role="navigation">
+    <li><%= span_if_current ts('Create a Token'), new_admin_api_path %></li>
+  </ul>
+
+  <table>
+    <thead>
+      <tr>
+        <td>Name</td>

```

Comment:
These `<td>`s on lines 11-15 should all be `<th scope="col">`, since they're headers for the information in the column. It's a good idea to avoid empty ones, so the one on line 15 can be named `<th scope="col">Actions</th>`.

These should probably be translation strings, too.